tests/loggers/test_comet.py

```diff
@@ -58,36 +38,49 @@ def test_comet_logger_online():
             experiment_name='experiment',
             api_key='key',
             workspace='dummy-test',
-            project_name='general'
+            project_name='general',
         )
 
         _ = logger.experiment
 
         comet_existing.assert_called_once_with(
-            api_key='key',
-            workspace='dummy-test',
-            project_name='general',
-            previous_experiment='test'
+            api_key='key', workspace='dummy-test', project_name='general', previous_experiment='test'
         )
 
         comet_existing().set_name.assert_called_once_with('experiment')
 
     with patch('pytorch_lightning.loggers.comet.API') as api:
-        CometLogger(
-            api_key='key',
-            workspace='dummy-test',
-            project_name='general',
-            rest_api_key='rest'
-        )
+        CometLogger(api_key='key', workspace='dummy-test', project_name='general', rest_api_key='rest')
 
         api.assert_called_once_with('rest')
 
 
+def test_comet_logger_experiment_name():
+    """Test that Comet Logger experiment name works correctly."""
+
+    api_key = "key"
+    experiment_name = "My Name"
+
+    # Test api_key given
+    with patch('pytorch_lightning.loggers.comet.CometExperiment') as comet:
+        logger = CometLogger(api_key=api_key, experiment_name=experiment_name,)
+
+        # The experiment object should not exists
```

Comment:
could you remove these type of comments, also in the other places below? the assertion below makes it very clear